rai/node/node.hpp

```diff
@@ -492,6 +492,7 @@ class node : public std::enable_shared_from_this <rai::node>
 	{
 		alarm.service.post (action_a);
 	}
+	void copy_with_compaction ();
     void send_keepalive (rai::endpoint const &);
```

Comment:
I avoided fixing existing indentation problems, since there's a patch already for that.